app/views/docs/rest.phtml

```diff
@@ -1,12 +1,337 @@
+    </tr>
+    <tr>
+        <td>file</td>
+        <td><code>[CHUNK-DATA]</code></td>
+        <td><code>[FILE-NAME]</code></td>
+        <td>Required, contains file chunk data.</td>
+    </tr>
+    <tr>
+        <td>permissions</td>
+        <td><code>[PERMISSION ARRAY]</code></td>
+        <td>n/a</td>
+        <td>Required, contains an array of permission strings about who can access the new file.</td>
+    </tr>
+    </tbody>
+</table>
+
+<p>While cURL and fetch are great tools to explore other REST endpoints, it's impractical to use for chunked file uploads because you need to split files into chunks.</p>
+
+<p>The multipart form data posted to file upload endpoints has the following format:</p>
+
+<div class="ide margin-bottom" data-lang="http" data-lang-label="HTTP">
+    <pre class="line-numbers"><code class="prism language-http" data-prism>POST /v1/storage/buckets/default/files  HTTP/1.1
+Host: [HOSTNAME_OR_IP]
+Content-Type: multipart/form-data; boundary=---WD9146A
+X-Appwrite-Project: demo-project
+content-range: bytes 10485760-12582912/12582912
+x-appwrite-id: 6369b0bc1dcf4ff59051
+
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="fileId"
+
+unique()
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="file"; filename="file.txt"
+Content-Type: application/octet-stream
+
+[CHUNKED-DATA]
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="permissions[]"
+
+read("user:627a958ded6424a98a9f")
+------WebKitFormBoundarye0m6iNBQNHlzTpVM--</code></pre>
+</div>
+
+<h2>Permissions</h2>
+
+<p>Appwrite SDKs have helpers to generate permission strings, but when using Appwrite without SDKs, you'd need to create the strings yourself.</p>
+
+<h3>Permission Types</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Permission String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Permission.read()</code></td>
+        <td><code>read("[PERMISSION_ROLE]")</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.create()</code></td>
+        <td><code>create("[PERMISSION_ROLE]")</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.update()</code></td>
+        <td><code>update("[PERMISSION_ROLE]")</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.delete()</td>
+        <td><code>delete("[PERMISSION_ROLE]")</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.write()</td>
+        <td><code>write("[PERMISSION_ROLE]")</code></td>
+    </tr>
+    </tbody>
+</table>
+
+<h3>Permission Roles</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Role String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Role.any()</code></td>
+        <td><code>"any"</code></td>
```

Comment:
For consistency, should we remove the double quotes here to match the Permission Types table?

```suggestion
        <td><code>any</code></td>
```